styles/icomoon/Read Me.txt

```diff
@@ -0,0 +1,7 @@
+Open *demo.html* to see a list of all the glyphs in your font along with their codes/ligatures.
```

Comment:
I dont think that we need this readme file
Because it is the readme about the fonts